scripts/map-randomizer.js

```diff
@@ -68,19 +69,24 @@
     shuffleArray(available);
     const selection = available.slice(0, amount);
 
-    if (!selection.length) return target.innerHTML = `
-          <div class="empty">
-            <div class="empty-icon">
-              <span class="icon icon-emoji"></span>
-            </div>
-            <p class="empty-title h5">Congratulations</p>
-            <p class="empty-subtitle">You've successfully selected no maps, you muppet</p>
+    if (!selection.length) {
+      target.innerHTML =
+        `<div class="empty">
+          <div class="empty-icon">
+            <span class="icon icon-emoji"></span>
           </div>
-        `;
+        <p class="empty-title h5">Congratulations</p>
+        <p class="empty-subtitle">You've successfully selected no maps, you muppet</p>
+        </div>`;
+      return;
+    }
 
-    target.innerHTML = '<ol>' + selection.map(function format(map) {
-      return `<li><span class="chip">${map.name}</span></li>`;
-    }).join('') + '</ol>';
+    target.innerHTML = selection.map((entry, idx) => {
+      return `<div class="img-container">
+                <img src="images/maps/${entry.name}.jpg" alt=${entry.name} class="map-img"/>
```

Comment:
In terms of accessibility, I would also recommend adding `aria-hidden="true"` the image to prevent double announcements of the map for the visually impaired. As a screen reader would read out image's alt tag, and the actual name as well.

Reply:
Oh forgot to mention, there are class names to make the images responsive; https://picturepan2.github.io/spectre/elements/media.html :D

Reply:
Yeah, I expected this comment. I shouldn't have been so lazy and just renamed them prior to creating the PR. Sorry.